homework11/Derkach.py

```diff
@@ -0,0 +1,21 @@
+import sqlite3
+
+with sqlite3.connect('Keyboard.db') as connection:
+    connection.execute('PRAGMA foreign_keys = 1')
+
+    cursor = connection.cursor()
+    cursor.execute(""" 
+        CREATE TABLE IF NOT EXISTS  Keyboard( 
+            keyboards INTEGER PRIMARY KEY AUTOINCREMENT,
+            name TEXT NOT NULL,
+            color TEXT NOT NULL,
+            model TEXT NOT NULL,
```

Comment:
This will cause an error, there is no need to put a comma on the last element in the list